lib/translator/common.h

```diff
@@ -20,16 +20,33 @@
 
 #include "absl/strings/string_view.h"
 #include "absl/types/span.h"
+
 #include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
 
 namespace translator {
 
-enum class StatusCode { kSuccess, kInvalidInput, kNoTranslation, kFailure };
+// Reports the status of a translation for internal use
+enum class StatusCode {
```

Comment:
nit: Maybe rename this to `TranslationState`? Also consider making it a private member type inside `Translation`, if it's no longer in the public API at all.

Reply:
Nevermind, this is used by downstream translation code.